valhalla/thor/triplegbuilder.h

```diff
@@ -10,15 +10,23 @@
 #include <valhalla/baldr/graphid.h>
 #include <valhalla/baldr/graphreader.h>
 #include <valhalla/baldr/pathlocation.h>
+#include <valhalla/meili/match_result.h>
 #include <valhalla/proto/trip.pb.h>
 #include <valhalla/sif/costfactory.h>
 #include <valhalla/thor/attributes_controller.h>
-#include <valhalla/thor/match_result.h>
 #include <valhalla/thor/pathinfo.h>
 
 namespace valhalla {
 namespace thor {
 
+// Allows you to trim shape for when you need to cut an edge for a discontinuity in map matching
```

Comment:
Should this be declared in meili?

Reply:
its used as part of the interface to triplegbuilder only. meili marks discontinuities on the edge segments